flutter/Makefile

```diff
@@ -48,6 +46,33 @@ debug_flags_windows=-c dbg --copt /Od --copt /Z7 --linkopt -debug
 .PHONY: android
 android: backend-bridge-android backends/tflite-android prepare-flutter
 
+.PHONY: android/apk
+android/apk: android
+	flutter clean
+	@# take results from build/app/outputs/flutter-apk/app-release.apk
+	flutter build apk
+
+.PHONY: ci/android/test_apk
+ci/android/test_apk: android/apk
+	@# take results from build/app/outputs/apk/androidTest/debug/app-debug-androidTest.apk
+	cd android && ./gradlew app:assembleAndroidTest
+	@# take results from build/app/outputs/apk/debug/app-debug.apk
+	cd android && ./gradlew app:assembleDebug -Ptarget=integration_test/first_test.dart
+
+.PHONY: docker/android/apk
+docker/android/apk: android/docker/image
+	@# if the build fails with java.io.IOException: Input/output error
+	@# remove file android/gradle/wrapper/gradle-wrapper.jar
+	MSYS2_ARG_CONV_EXCL="*" docker run --rm -it --user `id -u`:`id -g` -v $(CURDIR)/..:/mnt/project mlcommons/mlperf_mobile_flutter bash -c "cd /mnt/project/flutter && make prepare-flutter && make android/apk"
```

Comment:
This command doesn't utilize bazel cache.
I consider this command a proof that the docker container works both on CI and locally, and not a command that people should actually use to build the app.
A plan to add cache support while refactoring makefiles for https://github.com/mlcommons/mobile_app_open/issues/112.